tests/pages/firefox/welcome/page1.py

```diff
@@ -11,8 +11,13 @@ class FirefoxWelcomePage1(FirefoxBasePage):
 
     URL_TEMPLATE = '/{locale}/firefox/welcome/1/'
 
-    _monitor_button_locator = (By.ID, 'fxa-monitor-submit')
+    _monitor_primary_button_locator = (By.CSS_SELECTOR, '.primary-cta .js-monitor-button')
```

Comment:
I think you still need to update the actual test files that use these selector changes. Please push this branch to `run-integration-tests` to make sure things are passing.

Reply:
ah whoops! thank you. Looks like tests are passing now :+1: